tests/databricks/test_workflow.py

```diff
@@ -289,79 +291,6 @@ def test_create_workflow_with_arbitrary_extra_job_params(
     )
 
 
-@mock.patch("astro_databricks.operators.workflow.DatabricksHook")
-@mock.patch("astro_databricks.operators.workflow.ApiClient")
-@mock.patch("astro_databricks.operators.workflow.JobsApi")
-@mock.patch("astro_databricks.operators.workflow._get_job_by_name")
-@mock.patch(
-    "astro_databricks.operators.workflow.RunsApi.get_run",
-    return_value={"state": {"life_cycle_state": "RUNNING"}},
-)
-def test_create_workflow_with_arbitrary_extra_job_params(
```

Comment:
Why remove this?

Reply:
This was an exact duplicate test and pre-commit complained about it. We have the same test just above it.